### DISCUSSION TITLE
System tray on multiple monitors

### DISCUSSION THREAD

**Discussion by Kiburici** [valinet/ExplorerPatcher#2681] [Ideas]
Hello! Can you add an option to have the system tray on all taskbars? I have multiple monitors and always I get frustrated that I dont have the system tray on every taskbar, this should be an option from Microsoft, but their support for multiple monitors always been ignored, we didn't even had the clock at the launch of Win11... I really hope that this can be added to ExplorerPatcher and I really think that are many others that would love this option!

**Comment by eaglexk** [comment]
I also lacking taskbar pinned apps section on each of my monitors, same as notification area (system tray icons: network, volume, app notifications etc.). Is it possible to add an option to display them on each monitor? Windows never had this feature but it's quite useful. I know DisplayFusion supports it but this tool is not free and quite overwhelmed with other stuff.

**Comment by pyrates999** [comment]
You commented on a closed idea, so it has been decided already not to implement it.
